src/app/components/moj/moj-pagination/moj-pagination-item/moj-pagination-item.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-moj-pagination-item',
```

Comment:
Change this: selector: 'app-moj-pagination-item',
To this: selector: 'app-moj-pagination-item, [app-moj-pagination-item]',